### PR TITLE
add footer

### PR DESCRIPTION
**Pull request by ibrohim5009**

### CODE REVIEW

**Comment by ibrohim5009** [comment]
add footer

**Review by Muhammadali2oo6 — CHANGES_REQUESTED** [review]
yoqmadi very bad!!!!!

**Comment by ibrohim5009** [comment]
footer add

File: src/components/footer/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+          />
+          <p className="text-[#fff] mt-8 text-[13px]">
+            My service is not something I even think about because <br /> it is
+            always there as we agreed. The customer service is <br /> responsive
+            and knowledge needs for
+          </p>
+          <br />
+          <h1 className="text-[#fff]">
+            Follow us on <span className="text-[blue]">______</span>
+          </h1>
```
Comment (tolibjon1015) [inline src/components/footer/Footer.jsx:32]: span ga width berib rang bersang yaxshi bo'lar edi

File: src/components/footer/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+        </div>
+        <div className=" mt-4  ">
+          <h1 className="text-[white]">Get In Touch</h1>
+          <hr className="w-12 h-1 text-orange-300 mt-2 " />
+          <div className="input flex border-white w-[350px] h-[55px] mt-5 ">
+            <input
+              className="item w-[340px]  h-[55px] mt-2 outline-none border-none  text-[white]"
+              type="text"
+              placeholder="Type Your Mail"
+            />
```
Comment (tolibjon1015) [inline src/components/footer/Footer.jsx]: image
textni surib qo'y

File: src/components/footer/Footer.jsx [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+      </div>
+      <div className="w-[100%] h-[40px] bg-[#515050] grid grid-cols-2 justify-between">
+        <p className="text-[15px] text-[#fff] mt-2 pl-2 cursor-pointer">
+          Copyright 2022 Hostim. All Rights Reserved
+        </p>
+        <p className="text-[15px] text-[#fff] pl-[420px] mt-2 cursor-pointer">
+          Terms & Conditions | Privacy Policy
+        </p>
+      </div>
+    </div>
```
Comment (tolibjon1015) [inline src/components/footer/Footer.jsx]: image
bir tekkisda emas

**Review by tolibjon1015 — CHANGES_REQUESTED** [review]

File: src/components/footer/Footer.jsx (hunk 1 above)
Comment (ibrohim5009) [reply src/components/footer/Footer.jsx:32]: bunga width beril maydi

File: src/components/footer/Footer.jsx [hunk 4]
```diff
@@ -3,130 +3,140 @@ import { FaFacebookF } from "react-icons/fa";
...
-            Support Center
-          </a>
+            <a className="text-[#fff] list-none leading-8 " href="#">
+              Support Center
+            </a>
 
-          <a className="text-[#fff] list-none leading-8 " href="#">
-            Support Center
-          </a>
+            <a className="text-[#fff] list-none leading-8 " href="#">
```
Comment (xakimovotabek200) [inline src/components/footer/Footer.jsx]: nima a?

File: src/components/footer/Footer.jsx [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+    <div>
+      <div className="w-[100%] h-[350px] bg-[#33313D] p-6 grid grid-cols-4 justify-around">
+        <div>
+          <img
+            className="hostim object-cover"
+            src="https://hostim-html.themetags.com/assets/img/home3/logo-white.png"
+            alt=""
+          />
+          <p className="text-[#fff] mt-8 text-[13px]">
+            My service is not something I even think about because <br /> it is
```
Comment (xakimovotabek200) [inline src/components/footer/Footer.jsx]: cursor-pointer bolishi kerak edi?

File: src/components/footer/Footer.jsx [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+          </p>
+          <br />
+          <h1 className="text-[#fff]">
+            Follow us on <span className="text-[blue]">______</span>
+          </h1>
+          <div className="flex mt-6 gap-3  cursor-pointer">
+            <FaFacebookF className="bg-gradient-to-r from-purple-500 to-pink-500 w-[30px] h-[30px] rounded-[50%] p-[7px]" />
+            <FaFacebookF className="bg-gradient-to-r from-purple-500 to-pink-500 w-[30px] h-[30px] rounded-[50%] p-[7px]" />
+            <FaFacebookF className="bg-gradient-to-r from-purple-500 to-pink-500 w-[30px] h-[30px] rounded-[50%] p-[7px]" />
+            <FaFacebookF className="bg-gradient-to-r from-purple-500 to-pink-500 w-[30px] h-[30px] rounded-[50%] p-[7px]" />
```
Comment (xakimovotabek200) [inline src/components/footer/Footer.jsx]: hover

**Review by xakimovotabek200 — CHANGES_REQUESTED** [review]

**Review by samandar0806 — APPROVED** [review]

**Review by samandar0806 — APPROVED** [review]
ozgartir

File: src/components/footer/Footer.jsx [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+
+function Footer() {
+  return (
+    <div>
+      <div className="max-[414px]:grid bg-[#33313D] p-6 grid grid-cols-4 justify-around">
+        <div>
+          <img
+            className="hostim object-cover"
+            src="https://hostim-html.themetags.com/assets/img/home3/logo-white.png"
+            alt=""
```
Comment (akbarali0705) [inline src/components/footer/Footer.jsx]: cursor pointer emasmi

**Review by akbarali0705 — CHANGES_REQUESTED** [review]

File: src/components/footer/footer.css [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+  padding-right: 80px;
+  border-radius: 5px;
+}
+.item {
+  background: none;
+  /* padding-left: 62px; */
+}
+.hostim {
+  border: none;
+}
```
Comment (akbarali0705) [inline src/components/footer/footer.css]: nimaga css

File: src/components/footer/Footer.jsx [hunk 9]
```diff
@@ -0,0 +1,124 @@
...
+          />
+          <p className="text-[#fff] mt-8 text-[13px]">
+            My service is not something I even think about because <br /> it is
+            always there as we agreed. The customer service is <br /> responsive
+            and knowledge needs for
+          </p>
+          <br />
+          <h1 className="text-[#fff]">
```
Comment (xakimovotabek200) [inline src/components/footer/Footer.jsx:30]: media

**Review by xakimovotabek200 — CHANGES_REQUESTED** [review]

File: src/components/footer/Footer.jsx [hunk 10]
```diff
@@ -0,0 +1,124 @@
...
+            <img
+              className=" w-[72px] h-[30px] cursor-pointer"
+              src="	https://hostim-html.themetags.com/assets/img/visa.png"
+              alt=""
+            />
+            <img
+              className=" w-[72px] h-[30px] cursor-pointer"
+              src="	https://hostim-html.themetags.com/assets/img/visa.png"
+              alt=""
+            />
```
Comment (tolibjon1015) [inline src/components/footer/Footer.jsx]: image
mediani to'g'irla

**Review by tolibjon1015 — CHANGES_REQUESTED** [review]

File: src/components/footer/Footer.jsx (hunk 5 above)
Comment (ibrohim5009) [reply src/components/footer/Footer.jsx]: buldi

File: src/components/footer/Footer.jsx [hunk 11]
```diff
@@ -0,0 +1,124 @@
...
+          </div>
+        </div>
+      </div>
+      <div className="w-[100%] h-[40px] bg-[#515050] grid grid-cols-2 justify-between">
+        <p className=" text-[15px] text-[#fff] mt-2 pl-2 cursor-pointer">
+          Copyright 2022 Hostim. All Rights Reserved
+        </p>
+        <p className=" text-[#fff] pl-[420px] mt-2 cursor-pointer">
```
Comment (samandar0806) [inline src/components/footer/Footer.jsx]: padding left kop emasmi

**Review by samandar0806 — APPROVED** [review]

File: src/components/footer/footer.css [hunk 12]
```diff
@@ -0,0 +1,25 @@
...
+  position: absolute;
+  margin-left: -16px;
+  content: "//";
+}
+
+.input {
+  border: 1px solid #555;
+  width: 300px;
+  height: 62px;
+  padding-right: 80px;
```
Comment (samandar0806) [inline src/components/footer/footer.css:18]: qoida boyicha padding left 10px dan otib ketsa padding ni ozini ishlatish kerak

**Review by samandar0806 — CHANGES_REQUESTED** [review]

File: src/components/footer/LinkServives.jsx [hunk 13]
```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { ServisecData } from "./ServisecData";
+function LinkServives() {
+  return (
+    <div className="mt-5">
```
Comment (akbarali0705) [inline src/components/footer/LinkServives.jsx:5]: divga nimaga mt bergansan?

**Review by Ahmadullo21 — APPROVED** [review]

**Review by xakimovotabek200 — APPROVED** [review]

**Review by akbarali0705 — CHANGES_REQUESTED** [review]

**Review by akbarali0705 — APPROVED** [review]

**Review by Muhammadali2oo6 — APPROVED** [review]
good job

File: src/components/footer/Link.jsx [hunk 14]
```diff
@@ -0,0 +1,23 @@
...
+    <div className="mt-5 lg:pl-6">
+      <h1 className="text-[#fff] font-bold text-[18px]">Quick Links</h1>
+      <hr className="w-12 h-1 mt-2" />
+
+      {data.map((item) => {
+        return (
+          <div className="mt-5" key={item.id}>
+            <h4 className="text-[#fff] list-none leading-4" href={item.link}>
+              {item.title}
+            </h4>
```
Comment (akbarali0705) [inline src/components/footer/Link.jsx]: nimaga h4?

**Review by akbarali0705 — CHANGES_REQUESTED** [review]

**Review by Muhammadali2oo6 — APPROVED** [review]

**Pull request merged**